test/features/support/ui_authorization_authenticate.js

```diff
@@ -0,0 +1,146 @@
+const pactum = require("pactum");
+const { Given, When, Then, Before, After } = require("@cucumber/cucumber");
+const {
+  localhost,
+  requestTime,
+  validTransactionId,
+  validIndividualId,
+} = require("./helpers/helpers");
+
+let specUIAuthAuthenticate;
+let invalidTransactionId;
+let invalidIndividualId;
+
+const baseUrl = `${localhost}authorization/authenticate`;
+
+const responseSchema = {
+  type: "object",
+  properties: {
+    responseTime: {
+      type: "string",
+    },
+    response: {
+      type: "object",
+      properties: {
+        transactionId: {
+          type: "string",
+        },
+      },
+      additionalProperties: false,
+    },
+    errors: {},
+  },
+  additionalProperties: false,
+};
+
+const requestFunction = (transactionId, individualId) =>
+  specUIAuthAuthenticate.post(baseUrl).withBody({
+    requestTime: requestTime,
+    request: {
+      transactionId: transactionId,
+      individualId: individualId,
+      challengeList: [
+        {
+          authFactorType: "pin",
```

Comment:
```authFactorType``` should be validated. For now it allows to put every string value.
Please change it to allow only: 
![image](https://user-images.githubusercontent.com/119664031/216079632-7fff3377-62ed-4a34-985a-af8fd4d59501.png)